server/src/main/java/org/elasticsearch/index/mapper/IpFieldMapper.java

```diff
@@ -455,7 +462,75 @@ public BlockLoader blockLoader(BlockLoaderContext blContext) {
             if (hasDocValues()) {
```

Comment:
Should i check for `FieldExtractPreference.DOC_VALUES` here?

Reply:
No, I don't think we have to. This only seems to be used in geo based field types.

Reply:
You are right, sorry, i actually meant `FieldExtractPreference.STORED`.

Reply:
it would be nice to check, yes.

Reply:
For context - that's the user asking you to load from _source. If you ignore it you are ignoring their kind request. You are sure allowed, but you probably shouldn't ignore it.